sycl/source/stream.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <climits>
```

Comment:
Please, align with the layout described in https://github.com/intel/llvm/pull/10648 i.e. include standard library headers last and use empty line separator to avoid clang-format re-ordering.